src/ucp/core/ucp_ep.h

```diff
@@ -238,8 +242,12 @@ typedef struct ucp_ep_config {
         } rndv_send_nbr;
 
         struct {
-            /* Maximal size for eager short */
-            ssize_t         max_eager_short;
+            /* Maximal size for eager short when memtype mds present */
+            ssize_t             memtype_max_eager_short;
```

Comment:
maybe we can create new struct type like this:

typedef struct {
     ssize_t   memtype_on;
     ssize_t   memtype_off;
} ucp_memtype_thresh_t;

and then here:
ucp_memtype_thresh_t max_eager_short;
  
and reduce some code duplication. 
pass `const ucp_memtype_thresh_t*` to ucp_tag_is_valid_inline()